### ISSUE TITLE
Feature Request: webm

### ISSUE THREAD

**Issue by retroactively** [noonesimg/obsidian-audio-player#24]
Could you please add the support of webm? Because the core plugin of obsidian, audio recorder, outputs the webm formart.
If your can add this support, our work flow will be more fluent.
Thank you very much.

**Comment by RCheesley** [comment]
+1, also had this issue with the Obsidian recordings format!

**Comment by offbrands-studio** [comment]
+ 1 - Thank you for this plugin!

**Comment by WritwikRay** [comment]
+1 also would love this feature

**Comment by Archangeloi89** [comment]
Yes please that would be great since I would use it for obsidians internaly created Audios in webm format. 
Thanks alot.
